helm-charts/sep-service/values.yaml

```diff
@@ -0,0 +1,134 @@
+          path: /
+          pathType: Prefix
+          backend:
+            servicePort: "{{ .Values.service.containerPort }}"
+            serviceName: "{{ .Values.fullName }}-svc-{{ .Values.service.name }}"
+stellar:
+   anchor:
+      config: in-memory
+      app-config:
+         type: config-spring-property                # Activate [config-spring-property] module
+         settings: "app-config"                        # The location of the configuration data
+      data-access:
+         type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
+         settings: data-spring-jdbc-sqlite           # The location of the configuration data in this file.
+      logging:
+         type: logging-logback
+         settings: logging-logback-settings
+   app_config:
+      app:
+         stellarNetworkPassphrase: Test SDF Network ; September 2015
+         hostUrl: https://www.stellaranchordemo.com
+         languages: en
+         horizonUrl: https://horizon-testnet.stellar.org
+         assets: assets-test.json
+         jwtSecretKey: secret
+      sep1:
+         enabled: true
+         stellarFile: file:/config/stellar-wks.toml
+      sep10:
+         enabled: true
+         homeDomain: www.stellaranchordemo.com
+         clientAttributionRequired: false
+         clientAttributionAllowList: lobstr.co,preview.lobstr.co # use this if we want to white list
+         # clientAttributionDenyList: # use this if we want to black list.
+         authTimeout: 900
+         jwtTimeout: 86400
+         signingSeed: SBTBRGMPWNLXOZQQQXXZRJKCZPYTAZT5FC4VEAECF2HOBCONTSH4EXKS
+      sep12:
+         enabled: true
+         customerIntegrationEndpoint: https://anchor-reference-server-dev.stellar.org
+      sep24:
+         enabled: true
+         interactiveJwtExpiration: 3600
+         interactiveUrl: https://anchor-reference-server-dev.stellar.org/sep24/interactive
+      sep31:
+         enabled: true
+         feeIntegrationEndPoint: https://anchor-reference-server-dev.stellar.org
+         depositInfoGeneratorType: self # self or circle
+      sep38:
+         enabled: true
+         quoteIntegrationEndPoint: https://anchor-reference-server-dev.stellar.org
+      circle:
+         circleUrl: https://api-sandbox.circle.com
+         apiKey: secret # circle API key
+      payment-gateway:
+         circle:
+            name: "circle"
+            enabled: true
+         stellar:
+            enabled: false
+            name: "stellar"
+            horizonUrl: https://horizon-testnet.stellar.org
+            secretKey: secret # stellar account secret key
+      circle-payment-observer:
+         enabled: true
+         horizonUrl: https://horizon-testnet.stellar.org
+         stellarNetwork: TESTNET # TESTNET or PUBLIC
+         trackedWallet: all
+      event:
+         enabled: true
+         publisherType: kafka
+      kafka_publisher:
+         bootstrapServer: anchor-platform-kafka:9092
```

Comment:
How will kafka be deployed?

Reply:
it expects kafka to be deployed and provide the bootstrap broker (will soon be a list) as config parameter.   that being said, we will be providing some terraform example code that sets up msk along with anchor platform.